src/components/Crop.js

```diff
@@ -4,10 +4,12 @@ import { equals, is, update, remove } from 'ramda'
 import interact from 'interactjs'
 import { DeleteIcon, NumberIcon } from './Icons'
 
+import { areaNotAvailable } from '../utils'
+
 class Crop extends Component {
   static cropStyle = (coordinate) => {
     const {
-      x, y, width, height,
+      x, y, width, height, background, zIndex,
```

Comment:
this will output some internal data to user

Reply:
these properties is need for the functionality: 
- z-index allow to work to current box event 
- background to set a red color to see what happens.

all these properties are temporary